internal/handlers/registration_handler.go

```diff
@@ -0,0 +1,71 @@
+package handlers
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/redhatinsights/mbop/internal/store"
+	"github.com/redhatinsights/platform-go-middlewares/identity"
+)
+
+type registationCreateRequest struct {
+	UID *string `json:"uid,omitempty"`
+}
+
+func RegistrationHandler(w http.ResponseWriter, r *http.Request) {
+	b, err := io.ReadAll(r.Body)
+	if err != nil {
+		do500(w, "failed to read body bytes: "+err.Error())
+		return
+	}
+
+	var body registationCreateRequest
+	err = json.Unmarshal(b, &body)
+	if err != nil {
+		do400(w, "failed to unmarshal body: "+err.Error())
```

Comment:
I know we're not doing this consistently at the moment, but is it worth adding stdout on these helpers for catching errors in logs as well?

Reply:
Added a logger call in the `doError` function - so we get all errors logged.